tools/test_runner/kill_nodes.erl

```diff
@@ -0,0 +1,23 @@
+-module(kill_nodes).
+-export([main/0]).
+-export([main/1]).
+
+main() ->
+    main([]).
+
+main(Opts) ->
+    Nodes = opts_to_nodes(Opts),
+    Result = kill_nodes(Nodes),
+    io:format("kill_nodes: done ~p~n", [Result]),
+    erlang:halt().
+
+kill_nodes(Nodes) ->
+    [{Node, rpc:call(Node, init, stop, [])} || Node <- Nodes].
```

Comment:
Won't be really helpful if the nodes are frozen. But should work in most cases. :)